tests/testthat/test-r-to-pmml-df-code.R

```diff
@@ -136,3 +136,39 @@ test_that("When multiple expressions are present within a function call and
   test_utils_run_generate_pmml_test(code, expected_pmml)
 
 })
+
+test_that("Data frame code within an else expressions works", {
+  code <- '
+    a <- function(b, table) {
+     if(b == 1) {
+       return(1)
+     }
+     else {
+      return(table[table$c == b, ]$d)
+     }
+}
```

Comment:
The test should be made more robust by having an else if statement and each if statement should return a data frame access code.